sunkit_image/enhance.py

```diff
@@ -0,0 +1,122 @@
+"""
+This module contains functions that can be used to enhance the entire solar image.
+"""
+
+import numpy as np
+import scipy.ndimage as ndimage
+
+__all__ = [
+    "mgn",
+]
+
+
+def mgn(
+    data,
+    sigma=[1.25, 2.5, 5, 10, 20, 40],
+    k=0.7,
+    gamma=3.2,
+    h=0.7,
+    weights=None,
+    truncate=3
+):
+
+    """
+    Multi-scale Gaussian normalization.
+
+    Parameters
+    ----------
+    data : `numpy.ndarray`
+        Image to be transformed.
+    sigma : `list`, optional
+        Range of Gaussian widths to transform over.
+        Defaults to ``[1.25, 2.5, 5, 10, 20, 40]``.
+    k : `float`, optional
+        Controls the severity of the arctan transformation.
+        Defaults to 0.7
+    gamma : `float`, optional
+        The value used to calculate the  global gamma-transformed image.
+        Ideally should be between 2.5 to 4.
+        Defaults to 3.2
+    h : `float`, optional
+        Weight of global filter to Gaussian filters.
+        Defaults to 0.7
+    weights : `list`, optional
+        Used to weight all the transformed images during the calculation of the
+        final image. If not specificed, all weights are one.
+    width : `int`
+        An odd integer defining the width of the kernel to be convolved.
```

Comment:
what is width?  It is not in the function signature.

Reply:
Removed it was an error.